src/service/impl/departures/index.ts

```diff
@@ -111,7 +111,8 @@ export default (): IDeparturesService => {
             startTime,
             timeRange,
             filterByLineIds: favorites?.map(f => f.lineId),
-            ...limit
+            limitPerLine,
+            ...limit,
```

Comment:
Looks like `limitPerLine` will be overwritten by `limit.limitPerLine` here when favorites are enabled. We probably don't provide both favorites and limitPerLine right now, but if we do, would it make sense for the query param to take precedence over the `limit` object?

It could be done something like this maybe?

```ts
const limit = favorites
  ? {
      limitPerLine: limitPerLine ?? numberOfDepartures,
      numberOfDepartures: numberOfDepartures * 10
    }
  : {
      limitPerLine, 
      numberOfDepartures 
  };
```